packages/core/src/codewhispererChat/tools/toolShared.ts

```diff
@@ -7,6 +7,10 @@ import path from 'path'
 import fs from '../../shared/fs/fs'
 
 export const maxToolResponseSize = 30720 // 30KB
+export const readOnlyCommands: string[] = ['ls', 'cat', 'echo', 'pwd', 'which', 'head', 'tail']
+export const maxBashToolResponseSize: number = 1024 * 1024 // 1MB
+export const lineCount: number = 1024
+export const dangerousPatterns: string[] = ['|', '<(', '$(', '`', '>', '&&', '||']
```

Comment:
I think these are specific to executeBash?

Reply:
Correct. Moved to the executeBash.ts